BackofficeBundle/Resources/public/coffee/extendView/submitAdmin.coffee

```diff
@@ -20,10 +20,18 @@ extendView['submitAdmin'] = {
             context:
               button: viewContext.button
             success: (response) ->
-              new viewClass(viewContext.addOption(
-                html: response
-                submitted: true
-              ))
+              if($(".formCreation").data('type') != "listableCreation")
+                  new viewClass(viewContext.addOption(
+                    html: response
+                    submitted: true
+                  ))
+              else
+                  successMessage = $( response )[0].innerHTML
```

Comment:
@itkg-nanne that's not working. 
But i create a note to improve that by return only success message without form on next sprint.